### PR TITLE
replaced gorilla with melody

### PR DESCRIPTION
**Pull request by NikkyAI**
also added websocat command to the `matterbridge.tml.sample`

### CODE REVIEW

**Comment by NikkyAI** [comment]
this does **NOT** send backlog to the websockets on connection
I plan to add that in a followup PR

`api/stream` still consumes messages from the buffer
and accessing `api/messages` will clear the message buffer

websockets do not consume messages as messages are directly sent to the melody router

File: bridge/api/api.go [hunk 1]
```diff
@@ -33,6 +35,31 @@ func New(cfg *bridge.Config) bridge.Bridger {
...
+		message := config.Message{}
+		err := json.Unmarshal(msg, &message)
+		if err != nil {
+			b.Log.Errorf("failed to decode message from byte[] '%s'", string(msg))
+		}
+		b.handleWebsocketMessage(message)
+	})
+	b.mrouter.HandleConnect(func(session *melody.Session) {
+		greet := b.getGreeting()
+		bytes, err := json.Marshal(greet)
```
Comment (42wim) [inline bridge/api/api.go:50]: rename bytes to data so we don't have a collision with the `bytes` package

File: bridge/api/api.go [hunk 2]
```diff
@@ -83,7 +110,14 @@ func (b *API) Send(msg config.Message) (string, error) {
 	if msg.Event == config.EventMsgDelete {
 		return "", nil
 	}
-	b.Messages.Enqueue(&msg)
+	b.Log.Debugf("enqueueing message from %s on ring buffer", msg.Username)
+	b.Messages.Enqueue(msg)
+
+	bytes, err := json.Marshal(msg)
```
Comment (42wim) [inline bridge/api/api.go:116]: rename bytes to data so we don't have a collision with the bytes package

**Review by 42wim — CHANGES_REQUESTED** [review]

**Comment by 42wim** [comment]
some small changes and it can be merged

**Comment by NikkyAI** [comment]
rebased on master and rnamed the requested bits
(also one occurrence where i forgot to return in case of error)

File: bridge/api/api.go (hunk 1 above)
Comment (NikkyAI) [reply bridge/api/api.go]: done

File: bridge/api/api.go (hunk 2 above)
Comment (NikkyAI) [reply bridge/api/api.go]: done

**Pull request merged**